modules/eslint-plugin/src/rules/signals/with-state-no-arrays-at-root-level.ts

```diff
@@ -19,7 +34,7 @@ export default createRule<Options, MessageIds>({
     },
     schema: [],
     messages: {
-      [messageId]: `Wrap the array in an record or dictionary.`,
+      [messageId]: `withState should accept a record or dictionary as an input argument.`,
```

Comment:
What about using the following message here?
While reporting the error, the `property` needs to be provided.
For an example see `use-effects-lifecycle-interface`.

```suggestion
      [messageId]: `The property type `{{ property }}` is forbidden as the initial state argument, wrap the property in a record or dictionary.,
```